src/components/GithubLink/GithubLink.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Icon } from '../Icon/Icon';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { CSSTransition } from 'react-transition-group';
+
+import './GithubLink.scss';
+
+const GithubLink = ({ name, color, size, className }) => {
+  const [isShown, setIsShown] = useState(false);
+  const setTooltipVisibility = (boolean) => {
+    setIsShown(boolean);
+  };
+
+  return (
+    <div className="github">
+      <CSSTransition
+        in={isShown}
+        timeout={800}
+        classNames="alert"
```

Comment:
To keep naming consistent, how is this class name?
```suggestion
        classNames="tooltip"
```